packages/cli/scripts/build.mjs

```diff
@@ -49,3 +49,21 @@ copyFileSync(
   new URL('src/util/get-latest-version/get-latest-worker.js', repoRoot),
   new URL('get-latest-worker.js', distRoot)
 );
+
+writeFileSync(
+  new URL('vc.js', distRoot),
+  `#!/usr/bin/env node
```

Comment:
Any reason in particular why this is defined inline? How about we move it to a separate file which gets copied to `dist`?

Reply:
I was going to do that but the code was small and unlikely to change.

We also write a file here so its not unprecedented:

https://github.com/vercel/vercel/blob/6150fbe18ef7fb4a6591a87a40c4452b2c9923ce/packages/cli/scripts/build.mjs#L8-L14